src/modules/Nfs.ycp

```diff
@@ -352,6 +353,60 @@
 	return true;
     }
 
+    /*
+     * Supports(a, b) is true if a must be mounted before b.
+     * That is, "/" Supports "/any", "/foo" Supports "/foo/bar",
+     * but "/foo" does not Support "/foof"
+     */
+    boolean Supports(any entry_a, any entry_b) {
```

Comment:
As long as the toposort API is fixed (and I don't want to change that) there is no place to put the reversal logic. The function value `boolean(thing_t, thing_t)` dictates it.
We could define `DependsOn (a,b)` and then we would have to define `Supports(a, b) { return DependsOn(b, a) }`